resources/styles~300x300.css

```diff
@@ -129,4 +129,14 @@
 .circle_comfy {
     r: "35";
     fill: #10A30B;
-}
\ No newline at end of file
+}
+
+.tile-list-item {
+  height: 80;
```

Comment:
Currently, the style is set to display 3 numbers on the screen. Would it be possible to change this so it displays 2.5 numbers, or 3.5 numbers? This means that the number on the bottom is cut off, which gives the user the idea that there is more below and initiates scrolling (the alarm set feature on your Fitbit does this for example). Otherwise it looks like there are only three items to select from.

You can even get more fancier by adding a small rectangle at the bottom of the .gui file which has an opacity gradient. But this is not required for this PR